src/Components/Description.js

```diff
@@ -57,6 +57,20 @@ export default function Description() {
         </Col>
         <Col md='auto'></Col>
       </Row>
+      <Row>
+        <Col md='auto'></Col>
+        <Col className='md-8'>
+          <iframe width="560"
+            height="315"
+            src="https://www.youtube.com/embed/nGCEpUAnkSg?controls=0&autoplay=1&mute=1&modestbranding=1&loop=1"
+            title="YouTube video player"
+            frameBorder="0"
```

Comment:
Looks like this attribute is deprecated: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-frameborder

Reply:
Good catch, I'll remove this